client/src/components/Panel/Panel.scss

```diff
@@ -108,3 +108,12 @@ button.panel-heading-utils {
   margin: 20px 60px 20px 60px;
   border-top: 1px solid rgba($separatorColor, 0.5);
 }
+
+.panel-source-link {
+  font-weight: bold;
```

Comment:
Should use the centrally defined variables when setting font weights, `$heavyFontWeight` etc.

Reply:
Done!